packages/react-sdk/src/helpers/caching/contentTypes/readReceipt.ts

```diff
@@ -5,7 +5,7 @@ import {
 import { ContentTypeId } from "@xmtp/xmtp-js";
 import { parseISO } from "date-fns";
 import { z } from "zod";
-import isAfter from "date-fns/isAfter";
+import { isAfter } from "date-fns";
```

Comment:
the linter requires that there's only 1 import statement per dependency. please combine the `isAfter` and `parseISO` imports into 1 statement.